news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Community Mentorship Program Winter 2017 Signups Now Open"
+date: 2017-10-08 12:00:00 +0000
+---
+Have you ever opened up the editor and just looked at everything it offers in confusion? Do you have no idea where to even begin beatmapping? Worry no further, the Community Mentorship Program might be the place for you!
+
+![](/wiki/shared/news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open/banner.png)
+
+Since its inception in July of 2016, the program has gone through five cycles where experienced mappers teach aspiring new mappers everything there is to know about beatmapping, from the fundamentals of handling the editor to abstract and complex concepts of beatmap design, such as structure, flow, and many more!
+
+**This time around we are scouting for mentors and mentees for the game modes osu!, osu!catch and osu!taiko!** 
+
+This iteration of the program will be hosting three out of the four gamemodes of osu! - if you're interested here's a short timeline on this:
+- Mentor signups are open from 8th October 2017 until the 15th October 2017.
```

Comment:
Put `of` back between the ordinal day and the month. If you wanted to not use `of` then use cardinal numbers (remove the `th`, `nd`, and `st` suffixes and remove the `the` before the dates).

Assuming that you want to keep ordinal numbers in the dates, here is what I'm suggesting:
`- Mentor signups are open from the 8th of October 2017 until the 15th of October 2017.`
(for this line, there is also a missing `the` for the October 8th date)